envs/tensorrt-env.yaml

```diff
@@ -1,10 +1,13 @@
 packages:
 {% set py_ver = python | float %}
 {% if build_type == 'cuda' %}
+  - feedstock : cudatoolkit      #[cudatoolkit == "11.2"]
   - feedstock: libc_wrappers     #[x86_64 and (cudatoolkit == '11.0' or cudatoolkit == '11.2' or (py_ver < 3.8 and cudatoolkit == "10.2" ))]
 {% if py_ver < 3.8 %}
   - feedstock : cudnn
-  - feedstock : tensorrt
+    # Disable TensorRT temporarily.
```

Comment:
@jayfurmanek - we need to enable this if new versions of TRT are ready.

Reply:
yes we do.
The TRT update is: https://github.com/open-ce/tensorrt-feedstock/pull/24